### PR TITLE
Remove express probe and add strong-express-metrics probe

### PR DESCRIPTION
**Pull request by DanCunnington**
For strongloop parity, tested with adapter - all express tests pass and also independently using express. As this is for strong-supervisor only, no documentation has been added to the readme

### CODE REVIEW

File: probes/strong-express-metrics-probe.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  Probe.call(this, 'strong-express-metrics');
+}
+
+util.inherits(StrongExpressMetricsProbe, Probe);
+
+//This method attaches our probe to the instance of the strong-express-metrics module (target)
+StrongExpressMetricsProbe.prototype.attach = function(name, target) {
+  var that = this;
+  if( name != "strong-express-metrics" ) return target;
+  if(target.__ddProbeAttached__) return target;
```
Comment (sam-github) [inline probes/strong-express-metrics-probe.js]: random whitespace.... two lines, different `if(` spacing!

File: probes/strong-express-metrics-probe.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  target.__ddProbeAttached__ = true;
+
+  // Call the onRecord function and emit an appmetrics event
+  target.onRecord(function(record) {
+    am.emit('express:usage-record', record);
+  });
+  return target;
+}
+
+module.exports = StrongExpressMetricsProbe;
```
Comment (sam-github) [inline probes/strong-express-metrics-probe.js]: missing terminating newline

**Comment by sam-github** [comment]
LGTM, except for some formatting issues (using eslint causes this stuff to become non-issues). Also, I would encourage you to use standard format for git commit messages: http://chris.beams.io/posts/git-commit/

File: probes/strong-express-metrics-probe.js (hunk 1 above)
Comment (sam-github) [inline probes/strong-express-metrics-probe.js]: specifically, on has a SPC after `(`, and the other does not

**Comment by tobespc** [comment]
LGTM

**Comment by DanCunnington** [comment]
For #264

**Pull request merged**